app/assets/templates/auth/signup.html

```diff
@@ -1,4 +1,23 @@
-<div class="container">
+<nav class="navbar">
```

Comment:
instead of repeating over and over the navbar, why don't you guys create an [angular directive](https://docs.angularjs.org/guide/directive)? Here is an example from [mychois](https://github.com/WDI-HK-8/mychois/pull/19/files#diff-575b081007e3a8f1d2c3d14173af6568R1)